### PR TITLE
WINDUP-3488: Release Notes for MTR 1.0 release

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
WINDUP-3488: Release Notes for MTR 1.0 release
Resolves: https://issues.redhat.com/browse/WINDUP-3488
MTR 1.0

### CODE REVIEW

**Comment by apinnick** [comment]
Do not create a folder and master adoc for each version. This is what you should have:
/release-notes-mtr
- docinfo.xml
- master.adoc

Nothing else. No subfolders for MTR 1.0.

File: docs/topics/mtr-rn-new-features.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-{LC_PSN}/{LC_PSN}_release_notes-1.0/master.adoc
+
+:_content-type: CONCEPT
+[id="mtr-rn-new-features_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) 1.0 as compared to Migration Toolkit for Applications (MTA) 5.3.
```
Comment (sshveta) [inline docs/topics/mtr-rn-new-features.adoc:9]: This needs to be replaced by MTR

**Review by sshveta — CHANGES_REQUESTED** [review]

File: docs/topics/mtr-rn-new-features.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply docs/topics/mtr-rn-new-features.adoc:9]: @sshveta  I messed it up a few minutes ago. Please take a look now.

Comment (sbeskin-redhat) [reply docs/topics/mtr-rn-new-features.adoc:9]: @sshveta  No, wrong again, sorry.

Comment (sbeskin-redhat) [reply docs/topics/mtr-rn-new-features.adoc:9]: @sshveta  Now, please

File: docs/release-notes-mtr/topics/about-the-mta-operator.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * docs/wgetting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="about-the-mta-operator_{context}"]
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/about-the-mta-operator.adoc:6]: mta to be replaced by mtr

File: docs/release-notes-mtr/topics/about-the-mta-operator.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+= About the {ProductShortName} Operator
+
+You can install the web console on OpenShift Container Platform versions 4.9-4.11 with the Migration Toolkit for Applications Operator.
+
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/about-the-mta-operator.adoc:10]: Migration Toolkit for Runtimes

File: docs/release-notes-mtr/topics/about-the-web-console.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+:_content-type: CONCEPT
+[id="about-the-web-console_{context}"]
+= About the {WebName}
+
+
+ifdef::mtr[]
+The {WebName} for the {ProductName} allows a team of users to assess and prioritize migration and modernization efforts for a large number of applications. It allows you to group applications into projects for analysis and provides numerous reports that highlight the results.
+endif::[]
+
+ifdef::mta[]
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/about-the-web-console.adoc:14]: mta to be replaced by mtr

File: docs/release-notes-mtr/topics/about-tools.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="about-tools_{context}"]
+= About the tools
+
+The {ProductName} ({ProductShortName}) provides several tools to assist you in the various stages of your migration and modernization efforts. Review the details of each tool to determine which is right for your project.
+
+ifdef::mta[]
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/about-tools.adoc:11]: same here

File: docs/release-notes-mtr/topics/add-rule-to-mtr.adoc [hunk 6]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="add-rule-to-mta_{context}"]
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/add-rule-to-mtr.adoc:6]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/about-the-intro-to-mta-guide.adoc [hunk 7]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="about-the-intro-to-mta-guide_{context}"]
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/about-the-intro-to-mta-guide.adoc:6]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/add-rule-to-mta.adoc [hunk 8]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/add-rule-to-mta.adoc:3]: File name to be changed to mtr

File: docs/release-notes-mtr/topics/add-rule-to-mta.adoc [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+:_content-type: PROCEDURE
+[id="add-rule-to-mta_{context}"]
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/add-rule-to-mta.adoc:6]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/add-rule-to-mta.adoc [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
+A {ProductName} rule is installed by copying the rule to the appropriate {ProductShortName} folder. {ProductShortName} scans for rules, which are files with the `.windup.xml` extension in the following locations:
+
+* Directory specified by the `--userRulesDirectory` argument on the {ProductShortName} command line.
+* `<{ProductShortName}_HOME>/rules/` directory. `<{ProductShortName}_HOME>` is the directory where you install and run the {ProductName} executable.
+* `${user.home}/.{LC_PSN}/rules/` directory. This directory is created by {ProductShortName} the first time it is run. it contains rules, add-ons, and the {ProductShortName} log.
++
+[NOTE]
+====
+In a Windows operating system, the rules are located in `\Documents and Settings&#x5c;<USER_NAME>\.mta\rules\` or `\Users&#x5c;<USER_NAME>\.mta\rules\`.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/add-rule-to-mta.adoc:17]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/add-rule-to-mtr.adoc [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+
+A {ProductName} rule is installed by copying the rule to the appropriate {ProductShortName} folder. {ProductShortName} scans for rules, which are files with the `.windup.xml` extension in the following locations:
+
+* Directory specified by the `--userRulesDirectory` argument on the {ProductShortName} command line.
+* `<{ProductShortName}_HOME>/rules/` directory. `<{ProductShortName}_HOME>` is the directory where you install and run the {ProductName} executable.
+* `${user.home}/.{LC_PSN}/rules/` directory. This directory is created by {ProductShortName} the first time it is run. it contains rules, add-ons, and the {ProductShortName} log.
++
+[NOTE]
+====
+In a Windows operating system, the rules are located in `\Documents and Settings&#x5c;<USER_NAME>\.mta\rules\` or `\Users&#x5c;<USER_NAME>\.mta\rules\`.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/add-rule-to-mtr.adoc:17]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/cli-args.adoc [hunk 12]
```diff
@@ -0,0 +1,198 @@
...
+--source <SOURCE_1> <SOURCE_2>
+----
+
+The `--source` argument now provides version support, which follows the link:http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html[Maven version range syntax]. This instructs {ProductShortName} to only run the rulesets matching the specified versions. For example, `--source eap:5`.
+
+[WARNING]
+====
+When migrating to JBoss EAP, be sure to specify the version, for example, `eap:6`. Specifying only `eap` will run rulesets for all versions of JBoss EAP, including those not relevant to your migration path.
+
+See link:{ProductDocIntroTo{ProductShortName}GuideURL}/index#migration_paths_getting-started-guide[Supported migration paths] in _{IntroToMTABookName}_ for the appropriate JBoss EAP version.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/cli-args.adoc:157]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/cli-run.adoc [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+You can run {ProductShortName} against your application.
+
+.Procedure
+
+. Open a terminal and navigate to the `<{ProductShortName}_HOME>/bin/` directory.
+. Execute the `windup-cli` script, or `windup-cli.bat` for Windows, and specify the appropriate arguments:
++
+[source,terminal]
+----
+$ ./mta-cli --input /path/to/jee-example-app-1.0.0.ear \
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/cli-run.adoc:18]: Replace mta with mtr-cli

File: docs/release-notes-mtr/topics/cli-run.adoc [hunk 14]
```diff
@@ -0,0 +1,131 @@
...
+----
+
+[id="cli-bash-completion_{context}"]
+== {ProductShortName} {CLINameTitle} Bash completion
+
+The {ProductShortName} {CLIName} provides an option to enable Bash completion for Linux systems, allowing the {ProductShortName} command-line arguments to be auto completed by pressing the Tab key when entering the commands. For instance, when Bash completion is enabled, entering the following displays a list of available arguments.
+
+[source,terminal]
+----
+$ <{ProductShortName}_HOME>/bin/mta-cli [TAB]
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/cli-run.adoc:89]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/create-basic-xml-rule.adoc [hunk 15]
```diff
@@ -0,0 +1,173 @@
...
+when(condition)
+  perform(action)
+otherwise(action)
+----
+
+Create a file with the following contents, which is the basic syntax for XML rules.
+
+[IMPORTANT]
+====
+The XML file name must include the `.windup.xml` or `.mta.xml` extension. Otherwise, {ProductShortName} does not evaluate the new rule.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/create-basic-xml-rule.adoc:26]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/create-first-xml-rule.adoc [hunk 16]
```diff
@@ -0,0 +1,250 @@
...
+            <!-- Perform an action -->
+        </perform>
+      </rule>
+   </rules>
+</ruleset>
+----
++
+[NOTE]
+====
+The XML file name must include the `.windup.xml` or `.mta.xml` extension. Otherwise, {ProductShortName} does not evaluate the new rule.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/create-first-xml-rule.adoc:93]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/eclipse-creating-custom-ruleset.adoc [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+.Procedure
+
+. Click the *Rulesets* tab.
+. Click the Create Ruleset icon (image:repository-new.gif[Create ruleset icon]).
+. Select a project and a directory for the ruleset.
+. Enter the file name.
++
+[NOTE]
+====
+The file must have the extension `.windup.xml` or `.mta.xml`.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/eclipse-creating-custom-ruleset.adoc:22]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/eclipse-importing-custom-ruleset.adoc [hunk 18]
```diff
@@ -0,0 +1,23 @@
...
+
+:_content-type: PROCEDURE
+[id="eclipse-importing-custom-ruleset_{context}"]
+= Importing a custom ruleset
+
+You can import a custom ruleset into the {PluginName} to analyze your projects.
+
+.Prerequisites
+
+* Custom ruleset file with a `.windup.xml` or `.mta.xml` extension.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/eclipse-importing-custom-ruleset.adoc:13]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/eclipse-installing-plugin.adoc [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+
+ifdef::disconnected[]
+. On a computer with network access, navigate to the {ProductName} link:{{ProductShortName}DownloadPageURL}[download site] and download the `{IDEPluginFilename}` file.
+endif::[]
+. Launch Eclipse or CodeReady Studio.
+. From the menu bar, select *Help* -> *Install New Software*.
+. Next to the *Work with* field, click *Add*.
+. In the *Name* field, enter `{ProductShortName}`.
+ifdef::eclipse-code-ready-studio-guide[]
+. In the *Location* field, enter `\http://download.jboss.org/jbosstools/photon/stable/updates/mta/` and click *OK*.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/eclipse-installing-plugin.adoc:42]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/exclude-files-and-packages.adoc [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+
+You can exclude packages by either of the following methods:
+
+* Using the `--excludePackages` argument.
+* Specifying the packages in a file contained within one of the ignored locations. Each package should be included on a separate line, and the file must end in `.package-ignore.txt`. For example, see `<{ProductShortName}_HOME>/ignore/proprietary.package-ignore.txt`.
+
+[id="exclude-files_{context}"]
+== Excluding files
+
+{ProductShortName} can exclude specific files, such as included libraries or dependencies, during scanning and report generation. Excluded files are defined in a file with the `.mta-ignore.txt` or `.windup-ignore.txt` extension within one of the ignored locations.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/exclude-files-and-packages.adoc:24]: Replace mta with mtr.

File: docs/release-notes-mtr/topics/installing-web-console-on-openshift.adoc [hunk 21]
```diff
@@ -0,0 +1,86 @@
...
+. Log in to the OpenShift web console as a user with `project-admin-user` privileges.
+. Switch to the *Developer* perspective and click *+Add*.
+. In the *Add* view, click *Operator Backed*.
+. Click the *{DocInfoProductName}* Operator.
+. Click *Create*.
+endif::[]
+ifdef::ocp-42[]
+. Launch the OpenShift web console.
+. Click the *Import YAML* button in the upper-right corner of the web console.
+. Select *mta* from the *Project* list.
```
Comment (sshveta) [inline docs/release-notes-mtr/topics/installing-web-console-on-openshift.adoc:67]: Replace mta with mtr.

**Review by sshveta — CHANGES_REQUESTED** [review]

**Comment by sshveta** [comment]
@sbeskin-redhat I suggest searching for text "mta" and "applications" and then replacing them with mtr or "Runtimes" 
There are still lot of occurrences of mta in the MTR release doc.

File: docs/release-notes-mtr/topics/installing-web-console-on-openshift.adoc (hunk 21 above)
Comment (sbeskin-redhat) [reply docs/release-notes-mtr/topics/installing-web-console-on-openshift.adoc:67]: @sshveta It's just a technical glitch - all those hundreds of files have nothing to do with the RN. They are already fixed in another PR.
Only 4 files make up the RN: 
docs/release-notes-mtr/master.adoc
docs/topics/mtr-rn-new-features.adoc
docs/topics/mtr-rn-known-issues.adoc
docs/topics/mtr-rn-resolved-issues.adoc

**Pull request merged**